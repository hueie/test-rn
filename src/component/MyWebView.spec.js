import React from 'react';
import {shallow} from 'enzyme';
import MyWebView from './MyWebView'

describe('MyWebView', () => {
    describe('Rendering', () => {
        it('should contain WebView', () => {
            //If you test redering, before you did, you should check -- -u "Renewering Snapshot" 
            const component = shallow(<MyWebView></MyWebView>);
            expect(component).toMatchSnapshot();
        })
    })
    describe('Functionality', () => {

    })
    describe('Interaction', () => {

    })
})