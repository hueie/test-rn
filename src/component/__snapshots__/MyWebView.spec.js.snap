// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyWebView Rendering should contain WebView 1`] = `ShallowWrapper {}`;
